components/layout_2020/flow/inline/mod.rs

```diff
@@ -2408,25 +2414,26 @@ impl<'layout_data> ContentSizesComputation<'layout_data> {
     }
 
     fn line_break_opportunity(&mut self) {
+        self.pending_whitespace.min_content = Au::zero();
```

Comment:
This is white space that is hanging right? I would put a comment. Note that we don't always properly hang white space in layout yet.

Reply:
Yes, either hanging (e.g. `white-space: pre-wrap`) or removed (e.g. `white-space: normal`).
If the space needs to be included, then `commit_pending_whitespace()` should be called first.